Source/VMware.vSphereDSC/Tests/Integration/Configurations/VMHostSettings/VMHostSettings_Config.ps1

```diff
@@ -0,0 +1,69 @@
+<#
```

Comment:
I think this file is not needed, because you already added example config file in the ESXi Configs folder.

Reply:
Correct